lib/modules/apostrophe-schemas/index.js

```diff
@@ -977,7 +977,7 @@ module.exports = {
       converters: {
         string: function(req, data, name, object, field, callback) {
           object[name] = self.apos.launder.string(data[name], field.def);
-          if (field.min && (object[name].length < field.min)) {
+          if (field.required && field.min && (object[name].length < field.min)) {
```

Comment:
May have to look into this more and ensure it works as intended. But to do that, I need to disable the client side validations I believe. Will push an updated commit to this sometime Sunday.

Reply:
Yes, this was a snippet of code that made it in early in my attempt to improve this and I have to go over it again to make sure it's working as intended.